ivy/functional/backends/torch/norms.py

```diff
@@ -0,0 +1,24 @@
+import torch
+from typing import Optional, List
+
+from ivy.func_wrapper import with_unsupported_dtypes
+from . import backend_version
+
+
+@with_unsupported_dtypes({"0.11.0 and below": ("bfloat16", "float16")}, backend_version)
```

Comment:
And I guess the version should be "1.11.0" here as well.

Reply:
Updated :+1: